edx_notifications/server/web/static/edx_notifications/js/views/notification_pane_view.js

```diff
@@ -257,6 +256,41 @@ var NotificationPaneView = Backbone.View.extend({
             }
         );
     },
+    visitNotification: function(e) {
+        var messageId = $(e.currentTarget).find('span').data('msg-id');
+        var clickLink = $(e.currentTarget).find('span').data('click-link');
+
+        if (this.selected_pane === "unread") {
+            this.collection.url = this.endpoints.user_notification_mark_read.replace("/0", "/" + messageId);
```

Comment:
hrm. This seems like we'll need to fix the endpoints that are getting passed into the backbone app. This URL manipulation shouldn't be necessary. I'll look into it.